qwed_tax/middleware/gusto_interceptor.py

```
@@ -47,11 +50,19 @@ def process_ai_payroll_request(self, ai_generated_payload: Dict[str, Any]) -> Di
         # Verify deterministic logic via the QWED tax verification engine
         try:
             result = self.tax_verifier.verify_us_payroll(entry=payroll_entry)
+            if not isinstance(result, VerificationResult):
+                return {
+                    "status": "BLOCKED",
+                    "risk": "VERIFIER_ERROR",
+                    "reason": "Tax verifier returned an unexpected result type.",
+                    "execution_permitted": False,
+                }
         except Exception as exc:
+            _logger.exception("TaxVerifier raised an unexpected error: %s", exc)
```

Ruff (0.15.2)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)